### PR TITLE
Travis OSX testing

### PR DESCRIPTION
**Pull request by ChrisKeefe**
This Draft PR has been opened for the sole purpose of testing Travis's MacOS support with this project. Do not merge.

### CODE REVIEW

**Comment by thermokarst** [comment]
This travis config is getting out of hand!

**Comment by ChrisKeefe** [comment]
Sure is, @thermokarst. Travis officially does not support python/MacOS builds, and fails when trying to install python 3.6, which it seems to want to do by default...?

A few teams have been able to make it work, but I think I'm almost ready to give Github Actions a shot. You mentioned having a repo where you were tinkering with that somewhere?

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-language: python
+language: c
```
Comment (thermokarst) [inline .travis.yml:1]: here be dragons

**Comment by thermokarst** [comment]
> You mentioned having a repo where you were tinkering with that somewhere?

https://github.com/caporaso-lab/q2-phylogenomics/blob/master/.github/workflows/main.yml

I'm not 100% this'll work at the moment, but you could try to replicate the workflow linked above here in this repo.

File: .travis.yml (hunk 1 above)
Comment (ChrisKeefe) [reply .travis.yml:1]: This confirmed that `language: python` was causing failure before my "before_install" commands could run. I suspect that means Travis MacOS builds will explode whenever `language: python`.

I'll give your workflow a shot - hopefully that's a lower pain threshold than attempting to recreate Travis's python environment manually... :dragon: :dragon_face: :dragon:

**Comment by ChrisKeefe** [comment]
This approach is crashing on OSX with package resolution issues from Conda. I'm going to put this on ice for now, and start tinkering with github actions here: https://github.com/ChrisKeefe/q2-diversity-lib/pull/3

**Comment by ChrisKeefe** [comment]
Closed. Moving this repo to Github Actions: https://github.com/ChrisKeefe/q2-diversity-lib/pull/3

**Pull request closed without merging**